client/src/js/components/bhRenderOption.js

```diff
@@ -0,0 +1,32 @@
+angular.module('bhima.components')
+  .component('bhRenderOption', {
+    templateUrl : '/modules/templates/bhRenderOption.tmpl.html',
+    controller  : RenderOptionController,
+    bindings    : {
+      orientation         : '=', // two-way binding
```

Comment:
In the future, it would be better to use a combination of a one-way binding `<` and a callback `&`, as recommended by the AngularJS component documentation.  This is fine for this PR since you didn't know at the start.